mythril/leveldb/client.py

```diff
@@ -11,21 +11,21 @@
 
 # Per https://github.com/ethereum/go-ethereum/blob/master/core/database_util.go
 # prefixes and suffixes for keys in geth
-headerPrefix = b'h'         # headerPrefix + num (uint64 big endian) + hash -> header
-bodyPrefix = b'b'           # bodyPrefix + num (uint64 big endian) + hash -> block body
-numSuffix = b'n'            # headerPrefix + num (uint64 big endian) + numSuffix -> hash
-blockHashPrefix = b'H'      # blockHashPrefix + hash -> num (uint64 big endian)
-blockReceiptsPrefix = b'r'  # blockReceiptsPrefix + num (uint64 big endian) + hash -> block receipts
+header_prefix = b'h'         # header_prefix + num (uint64 big endian) + hash -> header
+body_prefix = b'b'           # body_prefix + num (uint64 big endian) + hash -> block body
+num_suffix = b'n'            # header_prefix + num (uint64 big endian) + num_suffix -> hash
+block_hash_prefix = b'H'      # block_hash_prefix + hash -> num (uint64 big endian)
+block_receipts_prefix = b'r'  # block_receipts_prefix + num (uint64 big endian) + hash -> block receipts
 # known geth keys
-headHeaderKey = b'LastBlock'  # head (latest) header hash
+head_header_key = b'last_block'  # head (latest) header hash
 # custom prefixes
-addressPrefix = b'AM'       # addressPrefix + hash -> address
+address_prefix = b'AM'       # address_prefix + hash -> address
 # custom keys
-addressMappingHeadKey = b'accountMapping'  # head (latest) number of indexed block
-headHeaderKey = b'LastBlock'  # head (latest) header hash
+address_mapping_head_key = b'account_mapping'  # head (latest) number of indexed block
+head_header_key = b'last_block'  # head (latest) header hash
```

Comment:
Same as above. 'LastBlock' should not be changed, That's how things are in ethereum's DB.